src/ts/components/MatchOverlay.tsx

```diff
@@ -69,6 +75,21 @@ const matchOverlay = <TPluginState extends IPluginState>({
     debounceShowMatch.current(true)
   }, [currentMatchId]);
 
+  const getOffsets = React.useMemo(() => ({ placement }: {placement: Placement}): [number | null | undefined, number | null | undefined] => {
+    const isTop = placement.indexOf("top") >= 0;
+    const isBottom = placement.indexOf("bottom") >= 0
+    if(referenceElement && currentRectIndex != undefined && (isTop || isBottom)){
+      const rects = referenceElement?.getClientRects();
+      const hoverRect = rects[currentRectIndex];
+      const lastRect = rects[rects.length-1]
+      const heightMultiplier = isBottom ? Math.max(0, rects.length - 1 - currentRectIndex) : currentRectIndex;
+      const y =  -hoverRect.height * heightMultiplier;
+      const x = hoverRect.left - lastRect.left
+      return [x, y - 3]
```

Comment:
We could define a yOffset constant, describe what it's doing and why, and refer to that here, `3`'s a bit magic (it was probably a bit magic before, too 😅 )

Reply:
Yeah it's a good point, I will make that change!